src/envo/envo-edit.owl

```diff
@@ -39732,6 +39732,19 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> <http://purl.obolibrar
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000066> <http://purl.obolibrary.org/obo/ENVO_00000015>))
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_01001497>))
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105001> (thermoplastic)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://www.gesamp.org/publications/guidelines-for-the-monitoring-and-assessment-of-plastic-litter-in-the-ocean>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105001> "A physical quality inhering in a bearer by virtue of the bearer's disposition to deform by heating."@en)
```

Comment:
There's a logical hole here - the definition doesn't restrict itself to plastics. Any bearer that can deform by heating would become a "thermoplastic", which is not the case - elastic substances can also deform by heating.

Reply:
We've looked into the definition of thermoplastic and it looks like the term "thermoplastic" is often used in the context of describing plastics but is not restricted to that alone.
Here are the reference we looked up: [Lexico Dictionary](https://www.lexico.com/en/definition/thermoplastic) | [Merriam Webster Dictionary](https://www.merriam-webster.com/dictionary/thermoplastic)

Reply:
@meichen-liu very interesting - it seems this term also refers to those things made plastic by heating, even if they weren't plastic before. 

So merging some of that content we have:

>A physical quality inhering in a bearer by virtue of the bearer's disposition to become plastic on heating and hardening again on cooling.

But because this fits the representation of a disposition, you may want to put it under BFO [disposition ](http://purl.obolibrary.org/obo/BFO_0000016)

>A disposition inhering in a bearer by virtue of the bearer's ability to become plastic on heating and hardening again on cooling.

Strangely, PATO also has a [disposition](http://purl.obolibrary.org/obo/PATO_0001727). Not sure why, best to ignore for now. 

There's also a way to skip the realizables (and qualities) altogether, by using the [capable of](http://purl.obolibrary.org/obo/RO_0002215) relation. 

So you'd say that a thermoplastic is capable of "plasticization by heat" and "hardening on cooling" etc. We'd have to create those processes of course. I don't think we can express the ability to repeat these processes using the axioms (but we  can note it in the textual definition). 

Which way would you like to go for your SDG 14 terms?

Reply:
@meichen-liu I think this comment thread can be resolved, as the code has since been updated accordingly.